python/tests/test_imports.py

```diff
@@ -1,8 +1,9 @@
+# ruff: noqa: F811 F401
```

Comment:
:question: What is this about?
:wrench: Please always provide in-code explanations for turning off any lints.